### DISCUSSION TITLE
Defining mutations

### DISCUSSION THREAD

**Discussion by olisand** [hodcroftlab/covariants#206] [General]
Hello,
Very nice resource !

This is probably a very naïve question. I am wondering where/how to collect the defining mutations for each variant from, and keep those lists updated.

Thanks and best wishes,

Olivier

**Comment by emmahodcroft** [comment]
Hi Olivier! Apologies for the delay - I was on holiday and am just catching up.

The defining mutations can be found in [clusters.py](https://github.com/hodcroftlab/covariants/blob/master/scripts/clusters.py) for each variant. These generally don't change much after defining a variant, unless I've gotten a mutation wrong.  I hope this helps!

**Reply by olisand** [reply]
Hello Emma,
Don't worry, there was no hurry.
Thank you for the answer ! So, you had to encode it yourselves, it is not available through an API or something like that, right ?
Indeed not a problem if they don't change much. And new variants don't appear too often, fortunately.

**Reply by emmahodcroft** [reply]
Hi Oliver, yes that's right, there's no API
